byzcoin/tx_pipeline.go

```diff
@@ -327,13 +330,26 @@ func (p *txPipeline) processTxs(txChan <-chan ClientTransaction, initialState *t
 		p.wg.Add(1)
 		defer p.wg.Done()
 		intervalChan := getInterval()
+		var txHashes [][]byte
+	leaderLoop:
 		for {
 			select {
 			case tx, ok := <-txChan:
 				if !ok {
 					log.Lvl3("stopping txs processor")
 					return
 				}
+				for _, txHash := range txHashes {
+					if bytes.Compare(txHash, tx.Instructions.Hash()) == 0 {
```

Comment:
This is hashing the instructions for each loop I think. Can we get that out of it ?